### DISCUSSION TITLE
OPENAUDIBLE_HOME on macOS Catalina

### DISCUSSION THREAD

**Discussion by wsm4826** [openaudible/openaudible#1096] [Q&A] [answered]
How do I set the environment variable OPENAUDIBLE_HOME on macOS Catalina to point to an external drive?

**Comment by openaudible** [comment]
Starting with macOS Catalina -- by default it takes zsh and hence you need to change in zprofile. Please follow below steps
Launch Terminal app

Create profile file:

touch ~/.zprofile
Open file in editor:

open ~/.zprofile

For example:

export OPENAUDIBLE_HOME=/FULL/_PATH

**Reply by HassoEccard** [reply]
Trying the newest update (3.7.4) on Ventura 13.3.1, the environment variable doesn't seem to work.i.e. It doesn't change the 'Preferences' (destination folder) value.

**Reply by HassoEccard** [reply, marked as answer]
Found it in another question - have to use File|Manage Libraries and open a'books.json' file.

**Comment by wsm4826** [comment]
thanks

Sent from my iPhone

On Apr 21, 2023, at 7:06 AM, stephen kenny ***@***.***> wrote:

﻿

Found it in another question - have to use File|Manage Libraries and open a'books.json' file.
